manim/__main__.py

```diff
@@ -72,10 +73,14 @@ def main():
 
         # elif args.cmd == "some_other_cmd":
         #     something_else_here()
+        # TODO: Maybe make use of this when I understand this?
 
     else:
         config.digest_args(args)
 
+        if args.livestream:
+            livestream()
```

Comment:
Currently, an error is raised when ending the stream because the entrypoint continues to work through the remainder of the script -- which is not necessary, we don't need to process an input file and render it.

```suggestion
            livestream()
            return
```

Reply:
> Currently, an error is raised when ending the stream because the entrypoint continues to work through the remainder of the script -- which is not necessary, we don't need to process an input file and render it.

Oh that's what the debugger was telling me. How did I not see through that